index.html

```diff
@@ -12,74 +12,130 @@
+              ><img src="images/Icon - Linkedin.png" alt="LinkedIn"
+            /></a>
           </li>
           <li>
-            <a href="#"><img src="images/Icon - Angelist.png" alt="Angelist" /></a>
+            <a href="#"
+              ><img src="images/Icon - Angelist.png" alt="Angelist"
+            /></a>
           </li>
           <li>
-            <a href="#"><img src="images/Icon - Twitter.png" alt="Twitter" /></a>
+            <a href="#"
+              ><img src="images/Icon - Twitter.png" alt="Twitter"
+            /></a>
           </li>
           <li>
-            <a href="#"><img src="images/Icon - Medium.png" alt="Medium" /></a>
+            <a href="#"
+              ><img src="images/Icon - Medium.png" alt="Medium"
+            /></a>
           </li>
         </ul>
-      </section>
+      </div>
+    </div>
   </div>
 
 
-  <section class="works">
-    <div class="grid-container">
+<section class="works"> 
+  <div class="grid-container">
+    
+      
+    <div class="second-desktop">
+      <h2 class="recents">My Recent Works</h2>
+      <div class="line"></div>
+    </div>
+       
 
+      
+        <div class="item1">
+            <img src="images/Img Placeholder.png" alt="placeholder" class="work-img">
 
-      <div class="item0">
-        <h2 class="recents">My Recent Works</h2>
-        <img src="images/Img Placeholder.png" alt="placeholder">
-      </div>
-
+            <div class="third-desktop">
+            <h2>Multi-Post Stories</h2>
+            <p class="stories">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                  <div class="lang list">
+            <ul>
+              <li><button>css</button></li>
+              <li><button>html</button></li>
+              <button>bootstrap</button>
+              <li><button>Ruby</button></li>
+            </ul>
+                  </div>
+                  <button class="see-project">See Project</button>
+          </div>
+        </div>
+      
 
 
-      <div class=" item1">
-        <h2>Multi-Post Stories</h2>
-        <p class="stories">A daily selection of privately personalized reads; no accounts or sign-ups required. has been
-          the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
-        </p>
-        <div class="lang list">
+      <div class="grid item2 mobile">
+        <h2>Project Art Printing Data</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+          standard</p>
+        <div class="lang">
           <ul>
-            <li><button>css</button></li>
             <li><button>html</button></li>
             <li><button>bootstrap</button></li>
             <li><button>Ruby</button></li>
           </ul>
+
         </div>
```

Comment:
- [ ] As the previous code reviewer mentioned, the first card in the Figma template should be the look of the card when the user hovers it, not the look of the first card. When the user hovers over the card the title, description, dark filter, and technologies cards should disappear and the button 'see project' should appear for the user to click and display the popup that you will create in the next projects. Kindly fix the hover behavior of the project cards as the previous code reviewer mentioned.

Reply:
- [ **OPTIONAL** ] Hi @henry-dura, it looks like you have fixed the styling issue for the first card as suggested by both the previous reviewer. However, I believe the hover effect should be applicable to all the project cards. I am marking this as optional but I highly recommend you apply those effects in all the cards to make them consistent and in a workable condition which will help you in further tasks 👍🏽 😃